Project2Store/FWC/app/src/main/res/values-v21/styles.xml

```diff
@@ -0,0 +1,9 @@
+<resources>
+
+    <style name="AppTheme.NoActionBar">
+        <item name="windowActionBar">false</item>
+        <item name="windowNoTitle">true</item>
+        <item name="android:windowDrawsSystemBarBackgrounds">true</item>
+        <item name="android:statusBarColor">@android:color/transparent</item>
```

Comment:
On your main screen the status bar at the top is covered over with white - it's likely because of this line that makes the status bar transparent, then one of your views with a white background is actually underneath the status bar and is showing through